### PR TITLE
Blog app - creating a data model

### PR DESCRIPTION
**Pull request by NeckerFree**
**Project requirements**
Be sure to reference the [ERD diagram](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd.png) from the sneak peek if you have any questions about table or column names or data relationships.

- Create a new app. Give it a meaningful name.
- Initialize your project with Git.
- Make sure that your project has Postgres database set up.
- Build your project schema.

  - Create and run the necessary migration files.
  - Table and column names should match the ERD diagram.
    - note: photo for users table should be a link to an image
  - Foreign keys should be included.
  - All columns that are foreign keys should have a corresponding index.

Blog_App_ER

### CODE REVIEW

File: db/migrate/20220712221741_create_posts.rb [hunk 1]
```diff
@@ -0,0 +1,14 @@
+class CreatePosts < ActiveRecord::Migration[7.0]
+  def change
+    create_table :posts do |t|
+      t.string :title
+      t.string :text
+      t.integer :comments_counter
+      t.integer :likes_counter
+      t.string :timestamps
+      t.references :users, null: false, foreign_key: true
```
Comment (brytebee) [inline db/migrate/20220712221741_create_posts.rb]: - [ ] **Required:** Following the given [ERD diagram](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd.png), the user table's foreign key has been given an alias of `author_id`, kindly implement this throughout all models that `belongs_to`  to the user. See this article for [more info.](https://medium.com/@pawlkris/activerecord-naming-conventions-and-how-to-alias-around-them-afd6b357d479)

**Review by brytebee — CHANGES_REQUESTED** [review]
Hi, @NeckerFree👋,

Good job so far coming so far with the progress you've made! 💪💪

## Highlights
You have done amazing in the following:
- The DB adapter uses PostgreSQL.✔️
- The app has all the required models.✔️
- There are no linter errors.✔️
 
However, there are some issues that you still need to work on to go to the next project but you are almost there!



# Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Please, remember to tag me in your question so I can receive the notification.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @brytebee in your question so I can receive the notification.
### _**Use @brytebee on slack for faster responses.**_


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: db/migrate/20220712221741_create_posts.rb (hunk 1 above)
Comment (NeckerFree) [reply db/migrate/20220712221741_create_posts.rb]: Thanks, @brytebee I saw the document you referenced but it is not enough to resolve the alias of that reference. But as you suggest that this is the correct way to resolve the alias, I going to explore more about how to solve this issue.

Comment (NeckerFree) [reply db/migrate/20220712221741_create_posts.rb]: I resolved the alias using the Add a reference column to a table command.

**Review by Mihndim2020 — APPROVED** [review]
# STATUS: APPROVED!!!  🎉 🎉 🎉 :champagne: :champagne: :champagne:
Hi @NeckerFree, 

### Good job !!!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Good commit messages
:heavy_check_mark: Descriptive Readme file

Your project is complete! Great work on making the changes requested by the previous reviewer. There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 :champagne: :champagne: :champagne:

Congratulations-Memes-FEATURED

## Optional suggestions

_Check the comments under the review._

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**